plutus-ledger-api/src/PlutusLedgerApi/V1/Credential.hs

```diff
@@ -42,10 +50,15 @@ instance PlutusTx.Eq StakingCredential where
         PlutusTx.&& c PlutusTx.== c'
     _ == _ = False
 
--- | Credential required to unlock a transaction output
+-- | Credentials required to unlock a transaction output.
 data Credential
-  = PubKeyCredential PubKeyHash -- ^ The transaction that spends this output must be signed by the private key
-  | ScriptCredential ValidatorHash -- ^ The transaction that spends this output must include the validator script and be accepted by the validator.
+  =
+    -- | The transaction that spends this output must be signed by the private key.
+    -- Hashed with /BLAKE2b-244/. 28 byte. See `Crypto.PubKeyHash`.
```

Comment:
I don't think we need to repeat this here since it's on the documentation of the inner type.